lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_const_border_radius/visitor.dart

```diff
@@ -0,0 +1,17 @@
+part of 'prefer_const_border_radius.dart';
+
+class _Visitor extends RecursiveAstVisitor<void> {
+  final _nodes = <InstanceCreationExpression>[];
+
+  Iterable<InstanceCreationExpression> get nodes => _nodes;
+
+  @override
+  void visitInstanceCreationExpression(InstanceCreationExpression node) {
+    super.visitInstanceCreationExpression(node);
+
+    if (node.constructorName.beginToken.lexeme == 'BorderRadius' &&
+        node.constructorName.endToken.lexeme == 'circular') {
```

Comment:
About that: let's have `node.staticType.name == 'BorderRadius' && node.constructorName.name.name == 'circular'` check instead since it's more stable than using tokens.